contracts/Protocol.sol

```diff
@@ -5,12 +5,11 @@ import "./spec_interfaces/IProtocol.sol";
 
 contract Protocol is IProtocol, Ownable {
 
-    // TODO - allow amending future upgrades - see https://github.com/orbs-network/orbs-ethereum-contracts/pull/192#discussion_r391179003
-
     struct DeploymentSubset {
         bool exists;
-        uint version;
+        uint nextVersion;
```

Comment:
i prefer using `uint256` since that is the explicit type and the one passed in the function header

Reply:
Agree

Reply:
Also in PR #25